Modix/Mappings/FilterParametersToInfractionSearchCriteriaMapper.cs

```diff
@@ -8,7 +8,7 @@ public static class FilterParametersToInfractionSearchCriteriaMapper
     {
         public static InfractionSearchCriteria ToInfractionSearchCriteria(this FilterParameter[] filterParameters)
         {
-            return new InfractionSearchCriteria()
+            return new InfractionSearchCriteria
```

Comment:
Personally, I prefer always having `()` for anything that executes code (except properties, of course). Anyone else?

Reply:
I'm 6 and half a dozen about this. Don't mind either way to be honest

Reply:
If you're not passing in anything to the constructor and you're using field initialisation it's entirely redundant.

Reply:
Sure, it's redundant. It just feels icky.